babel/messages/frontend.py

```diff
@@ -39,6 +39,48 @@
     from configparser import RawConfigParser
 
 
+def listify_value(arg, split=None):
```

Comment:
That sort of usage will never have been (how's that for a temporal form?) supported by Babel... At least not the optparse frontend.